mango-core/src/main/scala/org/bdgenomics/mango/converters/AlignmentRecordConverterGA4GH.scala

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
I prefer GA4GHAlignmentRecordConverter.scala instead of AlignmentRecordConverterGA4GH.scala, unless other people have strong preference. It may also be a good idea to instead us GA4GH as package path and then just call it AlignmentRecordCoverter.scala (ie mango-core/src/main/scala/org/bdgenomics/mango/converters/ga4gh/AlignmentRecordConverter.scala